buildSrc/src/main/java/org/elasticsearch/gradle/util/Pair.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+package org.elasticsearch.gradle.util;
+
+public class Pair<L, R> {
```

Comment:
There's no existing tuple class we can use, already on the classpath?

Reply:
Gradle has one but it's in an internal package so I opted to just write one since it's so trivial.